attendance/views.py

```diff
@@ -42,7 +42,16 @@ def user_check(request, username):
         return True
     else:
         return False
-
+        
+def last_month(this_month=datetime.now().month):
+	if(this_month > 0 and this_month <= 12):
```

Comment:
Add doc string for this function. Also move it to `utils` file, where are similar functions are and import it from there.